airbyte-workers/src/main/java/io/airbyte/workers/process/AsyncOrchestratorPodProcess.java

```diff
@@ -282,6 +287,14 @@ public void create(final Map<String, String> allLabels,
 
     }
 
+    LOGGER.info("getting metric client env var in async orchestrator pod process");
+    final EnvConfigs envConfigs = new EnvConfigs();
+    final String metricClient = envConfigs.getMetricClient();
+    LOGGER.info("metric client in async orchestrator pod process");
+    LOGGER.info(metricClient);
+
+    envVars.add(new EnvVar(EnvConfigs.METRIC_CLIENT, metricClient, null));
```

Comment:
Don't you need the Datadog related env vars here?

Reply:
I thought so but apparently not since this is working without it

Reply:
'working' as in, the metric client is showing up as a properly initialized datadog metric client